tests/network/vmi_istio.go

```diff
@@ -183,6 +185,43 @@ var _ = SIGDescribe("[Serial] Istio", func() {
 				}, tests.ContainerCompletionWaitTime, time.Second).Should(Succeed(), fmt.Sprintf("all containers should complete in source virt-launcher pod"))
 			})
 		})
+		Describe("SSH traffic", func() {
+			var bastionVMI *v1.VirtualMachineInstance
```

Comment:
There is `vmi` variable already taken for the one that has the proxy enabled. That's used throughout the suite.
So I wanted to differentiate it a little instead of using `vmi2` or alike, and if I'm not wrong, bastion is common term for a VM that provides acces to a cloud deployment

Reply:
> bastion is common term for a VM that provides acces to a cloud deployment

I made this assumption based on seeing it used in PSI and Azure, so it may not be significant sample to say it's common term. Let's just say I've seen it used like this

Reply:
Fair enough. I don't have a better suggestion.